webapp/views/wpt-results.js

```diff
@@ -477,6 +498,14 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
 
   constructor() {
     super();
+    this.openAmendMetadata = (i, node) => {
+      return () => {
+        const amend = this.shadowRoot.querySelector('wpt-amend-metadata');
```

Comment:
Nit: if you assign an `id` (e.g. `amend`) to `wpt-amend-metadata`, you could access it via `this.$.amend` which also saves the cost of querying the DOM every time.